src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  // height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  transition: transform $transition-time ease;
+
+  &__picture {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
```

Comment:
The use of `margin` property here could be improved. Instead of setting all four values, it would be better to only set the necessary ones. For example, if you only want to set the bottom margin, you can use `margin-bottom: 40px;`.